README.md

```diff
@@ -88,4 +88,12 @@ Start the script
 python3 main.py
 ```
 
+> Note: Brownie verification tooling might rewrite the imports in the source submission. It transforms relative paths to imported contracts into flat paths ('./folder/contract.sol' -> 'contract.sol'), which makes Diffyscan unable to find a contract for verification.
+
+For brownie based projects start script with flag:
```

Comment:
Maybe "for contracts whose sources were verified by brownie tooling"...? Because contracts from brownie based project could be verified in any other way as well